packages/cli/src/commands/start.js

```diff
@@ -15,6 +15,11 @@ export default {
 
     // Listen and show ready banner
     await nuxt.server.listen()
+
+    if (typeof process.send === 'function') {
```

Comment:
This `if` statement might be not needed. I don't know which versions of Node support it but it might be since a while ?

Reply:
Do we really want a pm2 specific feature to be part of the Nuxt core? I think this would be much better placed in a nuxt ready hook:
```js
// nuxt.config.js
export default {
  hooks: {
    ready() {
      process.send('ready')
    }
  }
}
```

Reply:
(I don't think it's pm2 specific but it helps pm2 anyway)
I will check where the hook is trigerred but yeah, it seems a better solution for sure !

Reply:
From node docs
> Child Node.js processes will have a process.send() method of their own that allows the child to send messages back to the parent.

Reply:
Due to this small clarification, we need `if` statement

Reply:
@dschewchenko Thanks for clarifications. I think we should end up with @pimlie proposal, so we'll not have to change Nuxt core. Can you confirm it works ? And the `listen` `hook` might be better than `ready` one, IDK 🙂

Reply:
Ok. I will update example to use hook, and delete changes in core